Plugins/GitHub3/UserInputPrompt.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Windows.Forms;
+
+namespace GitHub3
+{
+    public partial class UserInputPrompt : Form
+    {
+        public string UserInput { get; private set; } = "";
+
+        public UserInputPrompt(string title, string label, bool isPassword = false)
```

Comment:
I struggle to find where it used. Could you please point me to it?

Reply:
That is no more used and planned to be removed but kept it for the moment because it could be a configurable replacement of https://github.com/gitextensions/gitextensions/blob/master/GitUI/Script/SimplePrompt.cs